policyengine_us/parameters/gov/states/ct/tax/income/credits/property_tax/threshold/age.yaml

```diff
@@ -0,0 +1,15 @@
+description: Connecticut limits its property tax credit to filers at or above this age.
+values:
+  2017-01-01: 65
```

Comment:
the age limitation does not apply in 2022 and on so we need:
2022-01-01: 0

Reply:
the eligibility in general does not apply after 2021

Reply:
How about I return 'where (period < 2021, dependent_eligible | age_eligible, True)', in the eligibility function.. that way the years after 2021 are always aligible.. but the ones that come before still follow the same eligibility requirement

Reply:
we should make a parameter the is named age_eligbility_available and make it a bool parameter (see ut eitc parameter) and then include an if statement for if true - we check for age and if false - we do not